app/src/main/java/fr/free/nrw/commons/AboutActivity.kt

```diff
@@ -0,0 +1,244 @@
+        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
+        val aboutText = getString(R.string.about_license)
+        /*
+          We can then access all the views by just using the id names like this.
+          camelCasing is used with underscore characters being ignored.
+         */
+        binding!!.aboutLicense.setHtmlText(aboutText)
+
+        @SuppressLint("StringFormatMatches") val improveText =
+            String.format(getString(R.string.about_improve), Urls.NEW_ISSUE_URL)
+        binding!!.aboutImprove.setHtmlText(improveText)
+        binding!!.aboutVersion.text = applicationContext.getVersionNameWithSha()
+
+        Utils.setUnderlinedText(
+            binding!!.aboutFaq, R.string.about_faq,
+            applicationContext
+        )
+        Utils.setUnderlinedText(
+            binding!!.aboutRateUs, R.string.about_rate_us,
+            applicationContext
+        )
+        Utils.setUnderlinedText(
+            binding!!.aboutUserGuide, R.string.user_guide,
+            applicationContext
+        )
+        Utils.setUnderlinedText(
+            binding!!.aboutPrivacyPolicy, R.string.about_privacy_policy,
+            applicationContext
+        )
+        Utils.setUnderlinedText(
+            binding!!.aboutTranslate, R.string.about_translate,
+            applicationContext
+        )
+        Utils.setUnderlinedText(
+            binding!!.aboutCredits, R.string.about_credits,
+            applicationContext
+        )
+
+        /*
+          To set listeners, we can create a separate method and use lambda syntax.
+        */
+        binding!!.facebookLaunchIcon.setOnClickListener { view: View? ->
+            this.launchFacebook(
+                view
+            )
+        }
+        binding!!.githubLaunchIcon.setOnClickListener { view: View? ->
+            this.launchGithub(
+                view
+            )
+        }
+        binding!!.websiteLaunchIcon.setOnClickListener { view: View? ->
+            this.launchWebsite(
+                view
+            )
+        }
+        binding!!.aboutRateUs.setOnClickListener { view: View? ->
+            this.launchRatings(
+                view
+            )
+        }
+        binding!!.aboutCredits.setOnClickListener { view: View? ->
+            this.launchCredits(
+                view
+            )
+        }
+        binding!!.aboutPrivacyPolicy.setOnClickListener { view: View? ->
+            this.launchPrivacyPolicy(
+                view
+            )
+        }
+        binding!!.aboutUserGuide.setOnClickListener { view: View? ->
+            this.launchUserGuide(
+                view
+            )
+        }
+        binding!!.aboutFaq.setOnClickListener { view: View? ->
+            this.launchFrequentlyAskedQuesions(
+                view
+            )
+        }
+        binding!!.aboutTranslate.setOnClickListener { view: View? ->
+            this.launchTranslate(
+                view
+            )
+        }
```

Comment:
Since these methods all take the single `view` parameter, you can replace them with method references for a more succinct syntax
```suggestion
        binding!!.facebookLaunchIcon.setOnClickListener(::launchFacebook)
        binding!!.githubLaunchIcon.setOnClickListener(::launchGithub)
        binding!!.websiteLaunchIcon.setOnClickListener(::launchWebsite)
        binding!!.aboutRateUs.setOnClickListener(::launchRatings)
        binding!!.aboutCredits.setOnClickListener(::launchCredits)
        binding!!.aboutPrivacyPolicy.setOnClickListener(::launchPrivacyPolicy)
        binding!!.aboutUserGuide.setOnClickListener(::launchUserGuide)
        binding!!.aboutFaq.setOnClickListener(::launchFrequentlyAskedQuesions)
        binding!!.aboutTranslate.setOnClickListener(::launchTranslate)
```